app/src/main/java/com/powerhungers/fomezero/feature/registration/domain/model/Register.kt

```diff
@@ -0,0 +1,13 @@
+package com.powerhungers.fomezero.feature.registration.domain.model
+
+data class Register(
```

Comment:
```suggestion
data class RegistrationModel(
```
o que acha? na minha opniao fica mais claro que é um objeto de domain utilizada para o registro